festim/concentration/traps/neutron_induced_trap.py

```diff
@@ -7,7 +7,7 @@ class NeutronInducedTrap(ExtrinsicTrapBase):
     Class for neutron induced trap creation with annealing.
     The temporal evolution of the trap density is given by
 
-    dn_t/dt = phi*K*(1 - n_t/n_max) + A_0*exp(-E_A/(k_B*T))*n_t
+    dn_t/dt = phi*K*(1 - n_t/n_max) - A_0*exp(-E_A/(k_B*T))*n_t
```

Comment:
How is this related to the bug?

Reply:
Its not, just a small typo that needed fixing anyway

Reply:
Mb I thought that was the formulation itself. Shouldn't review PRs on my phone.....